src/graph.css

```
@@ -0,0 +1,48 @@
+.graph-container {
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 1rem;
+}
+
+.graph-node {
+  padding: 1rem 1.5rem;
+  border-radius: 999px;
+  background: rgba(255,255,255,0.08);
```

Stylelint (17.10.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)